.github/ISSUE_TEMPLATE/component_release_template.md

```diff
@@ -33,7 +33,7 @@ If including changes in this release, increment the version on `{{RELEASE_VERSIO
 - [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
 - [ ] All the tasks in this issue have been reviewed by the release owner.
 - [ ] Create, update, triage and label all features and issues targeted for this release with `v{{RELEASE_VERSION}}`.
-
+- [ ] Finalize the code and create the the release branch `{{RELEASE_VERSION}}` from the `{{RELEASE_VERSION_X}}` branch.
```

Comment:
Can you add a newline after this?

Thanks

Reply:
Fix it, thanks.